libafl/src/executors/inprocess.rs

```diff
@@ -621,6 +705,8 @@ mod windows_exception_handler {
         I: Input,
         Z: HasObjective<I, OF, S>,
     {
+        // TODO windows_remove_timeout()
+        // We can use DeleteTimerQueueEx, but you need a pointer to the timer_queue, which is not accessible from inproc_crash_handler.
```

Comment:
Can we make a somewhat hackish solution like storing the pointer thread local?

Reply:
Yes, I guess it's possible, but I'm undecided on which pointer to store. 
1. pointer to the `executor: Executor` or
2. pointer to the `timer_queue: HANLDE` 
3. or other thingy?

IMO, the pointer to the executor seems better, if we can tie `remove_timeout()` to the `Exectuor` then we can use like `executor->remove_timeout()` inside crash_handler

Reply:
I'd probably just do the `HANDLE`, if it's thread local you usually don't have more than a single pointer at this point. Buit either way is fine ;)

Reply:
I haven't tried, but the microsoft doc says the callback is run in another thread pool, so thread_local! might not work.
I will try passing the ptr to the executor to the callback :)